playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from '@playwright/test';
```

Comment:
why do we need this?

Reply:
This is the way the playwright documentation says to do imports, so I followed it... Not sure why it needs to be this way, I can try to find out if you think it would be valuable.